doc/magic_cards_notes.md

```diff
@@ -1142,8 +1143,6 @@ All commands are available before sealing. After the sealing acts as a Mifare Cl
 * Magic wakeup: `40(7)`, `43`
   * Backdoor read main block: `30xx+crc`
   * Backdoor write main block: `A0xx+crc`, `[16 bytes data]+crc`
-  * Read hidden block: `38xx+crc`
```

Comment:
UFUID chips don't have hidden blocks

Reply:
Suggested deleted lines 1145 and 1146 are in section `Mifare Classic > Mifare Classic USCUID > UFUID`, well above the proposed new USCUID-UL section which begins at line 1971.

Reply:
yes,  and are you saying that no `Mifare Classic > Mifare Classic USCUID`  has hidden blocks or?

Reply:
No, only the “cut down” varieties do not have the hidden blocks such as `UFUID` and `ZUID`. Looking at it now, code lines `1235` and `1236` should also be deleted.

Reply:
So,   it would need to have partion for those who has hidden blocks.

Reply:
It does, on lines `869-870` as general info, lines `1051-1052` for the `FUID` variant, lines `1313-1314` for the `GDM` variant, and lines `1382-1383` for the `GDCUID` variant. The UFUID and ZUID variants do not have hidden blocks, hence the request to remove those lines only.